client/packages/common/src/intl/locales/en/common.json

```diff
@@ -179,6 +179,7 @@
   "description.default-pack-size": "Default pack size",
   "description.entered": "The creation date",
   "description.expiry": "The closest expiry date of current stock on hand",
+  "label.date-of-death": "Date of death",
```

Comment:
i really thought we put sorting in one of the gh actions? alas...

Reply:
Not yet, the issue is still open I think